package.json

```diff
@@ -5,13 +5,16 @@
   "main": "index.js",
   "scripts": {
     "server_dev": "nodemon server --watch",
-    "react_dev": "webpack --mode development --watch"
+    "react_dev": "webpack --mode development --watch",
+    "seed": "node server/db/seed.js"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
     "express": "^4.17.1",
+    "i": "^0.3.6",
```

Comment:
Huh, I didn't know npm-i is a package. Did you mean to install this or was it a typo?

Reply:
Yea its not a package. I had another commit that removed that. Not sure why some commits pushed. Im looking into that right now.